README.md

```diff
@@ -10,7 +10,10 @@ The Go linter `paralleltest` checks that the t.Parallel gets called for the test
 paralleltest ./...
 ```
 
-To ignore missing calls to `t.Parallel` and only report incorrect uses of it, pass the flag `-i`.
+A few options can be activated by flag:
+
+* `-i`: Ignore missing calls to `t.Parallel` and only report incorrect uses of it.
+* `-ignoremissingsubtests`: Require that top-level tests specify `t.Parallel`, but don't require it in subtests (`t.Run(...)`).
```

Comment:
The flag naming here might be a bit of a mouthful, but I didn't want to pick another arbitrary one letter name like `-i` because longer term, that'll add to confusion (say more flags are added), and these flags will rarely be typed by human anyway.

In terms of `-ignoremissingsubtests` versus `ignore-missing-subtests` versus `ignore_missing_subtests`, I went with the first option because that seems to be the convention preferred by Go's Plan 9 style flag system (see `go help build` for some examples of this).